app/controllers/api/house_units_controller.rb

```diff
@@ -0,0 +1,106 @@
+module Api
+    class HouseUnitsController < ApplicationController
+  
+      # GET /api/house_units
+      def index
+        house_units = HouseUnit.all
+        render json: { data: house_units }, status: :ok
+      end
+  
+      # POST /api/house_units
+      def create
+        house_unit_params = create_request
+        is_house_unit_already_exist = false
+  
+        # Validate here

```

Comment:
in the next task, can we separate the Validation, to other file?

i.e. `validators/house_units/create_validator`, and called in controller.